ticdc/manage-ticdc.md

```diff
@@ -653,10 +667,69 @@ dispatchers = [
+
+| Event           | Type | Alias | Description         |
+| --------------- | ---- | -|--------------------------|
+| all dml         |      | |Matches all DML events       |
+| all ddl         |      | |Matches all DDL events         |
+| insert          | DML  | |Matches `insert` DML event      |
+| update          | DML  | |Matches `update` DML event      |
+| delete          | DML  | |Matches `delete` DML event      |
+| create schema   | DDL  | create database |Matches `create database` event |
+| drop schema     | DDL  | drop database  |Matches `drop database` event |
+| create table    | DDL  | |Matches `create table` event    |
+| drop table      | DDL  | |Matches `drop table` event      |
+| rename table    | DDL  | |Matches `rename table` event    |
+| truncate table  | DDL  | |Matches `truncate table` event  |
+| alter table     | DDL  | |Matches `alter table` event, including all clauses of `alter table` and `create index`/`drop index`   |
+| add table partition    | DDL  | |Matches `add table partition` event     |
+| drop table partition    | DDL  | |Matches `drop table partition` event     |
+| truncate table partition    | DDL  | |Matches `truncate table partition` event     |
+| create view     | DDL  | |Matches `create view`event     |
+| drop view     | DDL  | |Matches `drop view` event     |
+
+- `ignore-sql`: the DDL statements to be ignored. This parameter accepts an array of strings, in which you can configure multiple regular expressions. This rule only applies to DDL events.
+- `ignore-delete-value-expr`: this parameter accepts a SQL expression. This rule only applies to delete DML events with the specified value.
+- `ignore-insert-value-expr`: this parameter accepts a SQL expression. This rule only applies to insert DML events with the specified value.
+- `ignore-update-old-value-expr`: this parameter accepts a SQL expression. This rule only applies to update DML events whose old value contains the specified value.
+- `ignore-update-new-value-expr`: this parameter accepts a SQL expression. This rule only applies to update DML events whose new value contains the specified value.
+
+> **Note:**
+>
+> - When TiDB updates a value in the column of the clustered index, TiDB splits a UPDATE event into a DELETE event and an INSERT event. TiCDC does not identify such events as an UPDATE event and thus cannot correctly filter out such events.
+> - When you configure a SQL expression, make sure all tables that matches `matcher` contain the columns specified in the SQL expression. Otherwise, the replication task cannot be created. In addition, if the table schema changes during the replication, which results in a table no longer containing a required column, the replication task enters an error state that cannot be resumed automatically. In such a situation, you must manually modify the configuration and resume the task.
```

Comment:
```suggestion
> - When you configure a SQL expression, make sure all tables that matches `matcher` contain all the columns specified in the SQL expression. Otherwise, the replication task cannot be created. In addition, if the table schema changes during the replication, which results in a table no longer containing a required column, the replication task fails and cannot be resumed automatically. In such a situation, you must manually modify the configuration and resume the task.
```